Code/Tools/AssetProcessor/native/utilities/ProductOutputUtil.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <native/utilities/ProductOutputUtil.h>
+#include <QString>
+#include <AzCore/std/containers/vector.h>
+#include <AzFramework/IO/FileOperations.h>
+#include <native/utilities/UuidManager.h>
+
+namespace AssetProcessor
+{
+    AZStd::string ProductOutputUtil::GetPrefix(AZ::s64 scanfolderId)
+    {
+        return AZStd::string::format("(%" PRIu64 ")", scanfolderId);
```

Comment:
Nitpick: This might cause compile warnings (and then errors) on Linux, you're implicitly converting from a signed value to an unsigned string format value.